cli/src/funTest/kotlin/OrtMainTest.kt

```diff
@@ -56,54 +56,51 @@ class OrtMainTest : StringSpec() {
         "Activating only Gradle works" {
             val inputDir = File(projectDir, "gradle")
 
-            val runResult = runMain(
+            val stdout = runMain(
                 "analyze",
                 "-m", "Gradle",
                 "-i", inputDir.path,
                 "-o", File(outputDir, "gradle").path
             )
-            val iterator = runResult.stdout.iterator()
+            val iterator = stdout.iterator()
             while (iterator.hasNext()) {
                 if (iterator.next() == "The following package managers are activated:") break
             }
 
-            runResult.exitCode shouldBe 0
```

Comment:
Why do you not test the exit codes anymore?

Reply:
Because `clikt`'s `parse()` throws `UsageError` instead.